web/tests/functional/report_viewer_api/__init__.py

```diff
@@ -114,7 +123,11 @@ def setup_class_common(workspace_name):
     print("Third analysis of the test project was successful.")
 
     # Let's run the second analysis and updat the same run.
-    codechecker_cfg['checkers'] = ['-d', 'core.StackAddressEscape']
+    codechecker_cfg['checkers'] = ['-d', 'core.StackAddressEscape',
+                                   '-d', 'unix.Malloc',
```

Comment:
Whats the reason behind this? Why did you disable MallocChecker?

Reply:
Earlier the meaning of `--disable` flag was to actually disable the checker in the `clang` invocation. Now the meaning of `--disable` flag is to hide the results of the given checker. So after this patch `core.StackAddressEscape` is actually running, even if disabled, but its reports are hidden by post-processing. This caused `unix.Malloc` checker to report its own diagnostics.